interface/modules/custom_modules/oe-module-claimrev-connect/public/index.php

```diff
@@ -0,0 +1,57 @@
+<?php
+    $tab="home";
+?>
+<html>
+    <head>
+        <link rel="stylesheet" href="../../../../../public/assets/bootstrap/dist/css/bootstrap.min.css">
+    </head>
+<title>ClaimRev Connect - Home</title>
+<body>
+    <div class="row"> 
+        <div class="col">
+            <?php
+                include '../templates/navbar.php';
```

Comment:
Generally use require in these situations so script will break if this file does not exist